File: android/trunk/src/com/usedparts/domain/AccountInfo.java
package com.usedparts.domain;

import com.fasterxml.jackson.annotation.JsonProperty;

public class AccountInfo {

    @JsonProperty("email")
    public String email;

    public String password;
    public String phone;
    public int organizationType;

    public int id;

    @JsonProperty("name")
    public String name;
    @JsonProperty("offers")
    public int offersCount;
    @JsonProperty("cars")
    public int carsCount;
    @JsonProperty("vcurr")
    public double balance;
}

File: wp/trunk/UsedParts.Domain/AccountInfo.cs
using Newtonsoft.Json;

namespace UsedParts.Domain
{

    public class AccountInfo
    {
        [JsonProperty(PropertyName = "id")]
        public int Id { get; set; }

        public string Email { get; set; }
        public string Password { get; set; }
        public string Phone { get; set; }
        public int OrganizationType { get; set; }

        [JsonProperty(PropertyName = "name")]
        public string Name { get; set; }

        [JsonProperty(PropertyName = "offers")]
        public int OffersCount { get; set; }

        [JsonProperty(PropertyName = "cars")]
        public int CarsCount { get; set; }

        [JsonProperty(PropertyName = "vcurr")]
        public double Balance { get; set; }
    }

}
